types/global.ts

```diff
@@ -1,31 +1,33 @@
 import Hippy from './hippy';
 
```

Comment:
If you have an import or an export at the root level of a TypeScript file then it creates a local scope within that file.